tests/phpunit/tests/db.php

```diff
@@ -1753,6 +1912,10 @@ public function test_escape_and_prepare( $escape, $sql, $values, $incorrect_usag
 		$actual = $wpdb->prepare( $sql, $values );
 
 		$this->assertSame( $expected, $actual );
+
+		if ( is_string( $incorrect_usage ) && array_key_exists( 'wpdb::prepare', $this->caught_doing_it_wrong ) ) {
+			$this->assertStringContainsString( $incorrect_usage, $this->caught_doing_it_wrong['wpdb::prepare'] );
```

Comment:
See above regarding multiple assertions.

Reply:
I've removed this one, as I've not changed anything in `data_escape_and_prepare()`.